compiler-core/src/parse/snapshots/gleam_core__parse__tests__type_angle_generics_definition_error.snap

```diff
@@ -0,0 +1,25 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "\ntype Maybe<A> {\n    Some(A)\n    None\n}\n"
+---
+----- SOURCE CODE
+
+type Maybe<A> {
+    Some(A)
+    None
+}
+
+
+----- ERROR
+error: Syntax error
+  ┌─ /src/parse/error.gleam:2:11
+  │
+2 │ type Maybe<A> {
+  │           ^ I was expecting `(` here.
+
+Instead of angle brackets, Gleam uses parentheses for generic types:
+
+    List(String)
+    Result(Int, Error)
```

Comment:
Same here, it would be great to show their actual type, so here we would show this:

```gleam
type Maybe(a) {
```